VideoLocker/res/values/strings.xml

```diff
@@ -450,6 +448,8 @@
     <string name="title_lecture_complete">You\'ve reached the end of this video.</string>
     <!-- Alert dialog prompt at end of video. -->
     <string name="msg_lecture_complete">Would you like to watch the next video?</string>
+    <!-- Accessible content description of cancel button to cancel video download -->
+    <string name="cancel">Cancel</string>
```

Comment:
I don’t think there is any way to declare which locales we support. The framework just tries to get the string from the current locale if available, otherwise it takes the base string.
> On 11-Sep-2015, at 9:54 pm, Akiva Leffert <notifications@github.com> wrote:
> 
> In VideoLocker/res/values/strings.xml <https://github.com/edx/edx-app-android/pull/354#discussion_r39291485>:
> 
> > @@ -450,6 +448,8 @@
> >      <string name="title_lecture_complete">You\'ve reached the end of this video.</string>
> >      <!-- Alert dialog prompt at end of video. -->
> >      <string name="msg_lecture_complete">Would you like to watch the next video?</string>
> > +    <!-- Accessible content description of cancel button to cancel video download -->
> > +    <string name="cancel">Cancel</string>
> Presumably, it will only show translations for locales we claim to support? If we claim to support a language then we should be fine showing the built in translation for that string?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/354/files#r39291485>.
>